cluster/kubevirtci.sh

```diff
@@ -12,18 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.19'}
-
-KUBEVIRTCI_VERSION="8f48705333a7b9c4c91cf8a0b96b40bb54ef6c8d"
-export KUBEVIRTCI_TAG=2103111738-8f48705
```

Comment:
I think that kubevirtci needs this.

Reply:
`KUBEVIRTCI_VERSION` is not in use in kubevirtci. It was used here to checkout a specific commit. I replaced it with checking out the tag.

Reply:
right, so I think we should alsop drop hack/bump-kubevirtci.sh